src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,107 @@
+/* #region cards */
+
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  width: 1024px;
+  margin: 0 auto;
+  padding: 50px 40px;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+
+  &__review,
+  &__product-price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__img {
+    margin-bottom: 40px;
+    width: 160px;
+    height: 134px;
+    align-self: center;
+  }
+
+  &__product-name {
+    color: $text-important-color;
+    font-size: 12px;
+    font-weight: 500;
+    margin: 0 0 4px;
+    line-height: 18px;
+  }
+
+  &__product-code {
+    color: $text-additional-color;
+    font-weight: 400;
+    margin: 0 0 16px;
+  }
+
+  &__review {
+    margin-bottom: 24px;
+
+    &-rate {
+      color: $text-important-color;
+    }
+  }
+
+  &__product-price {
+    margin-bottom: 16px;
+
+    &-text {
```

Comment:
use double `--` for modifiers

Reply:
The linter doesnt let me pass test when I change it to double dash, error:
Tag doesn't have modifiable class [card__product-price--text]  (modifiable-class)

Reply:
Did you use this modifier class in HTML markup?

Reply:
Yes